### DISCUSSION TITLE
Shadow a xrdp session

### DISCUSSION THREAD

**Discussion by HottuDoku** [neutrinolabs/xrdp#2448] [Q&A] [answered]
Hi,

one of my WHO users has ran into some system issue and requested our help. That person is using xrdp to remote to our server and I've been wondering if its possible to somehow remote (VNC and such) to that xrdp session, essentialy shadow it like you can on Windows Remote Server.

**Comment by matt335672** [comment, marked as answer]
There's nothing bullt in to XRDP to support this, at least not that I'm aware of.

Simplest generic way is probably for the user to run x11vnc, and for you to connect directly to that. The desktop you're using will offer its own sharing capabilities.
